src/wallet/rpcdump.cpp

```diff
@@ -1063,6 +1063,7 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
             "2. options                 (json, optional)\n"
             "  {\n"
             "     \"rescan\": <false>,         (boolean, optional, default: true) Stating if should rescan the blockchain after all imports\n"
+            "     \"rescanUpdate\": <false>,   (boolean, optional, default: true) Stating if rescan should notify existent transactions\n"
```

Comment:
Remove `<false>`, these `<false>` and `<true>` will be removed IMO.

Reply:
See: https://github.com/bitcoin/bitcoin/blob/364da2c52942ebfe92d1d1495a309a8a9dd18ef3/src/wallet/rpcwallet.cpp#L3060